src/argilla/server/schemas/v1/datasets.py

```diff
@@ -148,13 +158,46 @@ class RatingQuestionSettings(BaseModel):
     )
 
 
+class LabelSelectionQuestionSettingsOption(BaseModel):
+    value: constr(
+        min_length=LABEL_SELECTION_VALUE_MIN_LENGHT,
+        max_length=LABEL_SELECTION_VALUE_MAX_LENGHT,
+    )
+    text: constr(
+        min_length=LABEL_SELECTION_TEXT_MIN_LENGTH,
+        max_length=LABEL_SELECTION_TEXT_MAX_LENGTH,
+    )
+    description: Optional[
+        constr(
+            min_length=LABEL_SELECTION_DESCRIPTION_MIN_LENGTH,
+            max_length=LABEL_SELECTION_DESCRIPTION_MAX_LENGTH,
+        )
+    ] = PydanticField(None, max_length=100)
```

Comment:
We can remove the `PydanticField` definition since we're using `constr`, right?
```suggestion
    ]
```

Reply:
yes! I forgot to push that. I've added some unit tests also to validate the min/max length for the label selection question option attributes.